contracts/Strategy.sol

```diff
@@ -105,6 +158,119 @@ contract Strategy is BaseStrategy {
             uint256(int256(supply) + delta);
         uint256 newSupplyRate = cToken.getSupplyRate(newUtilization) *
             SECONDS_PER_YEAR;
-        return newSupplyRate;
+
+        uint rewardRate = getRewardAprForSupplyBase(getPriceFeedAddress(comp), delta);
+
+        return newSupplyRate + rewardRate;
+    }
+
+    function _tend() internal override {
+        // claim rewards, sell rewards, reinvest rewards
+        _claimCometRewards();
+        if(tradeFactory == address(0)) {
+            _sellRewards();
+        }
+        _invest();
```

Comment:
Probably want to put a check for if(balanceOfAsset() == 0) return through this flow, since if tradeFactory is set it will likely be 0.

Probably makes the most sense in _depositToComet() ?